### PR TITLE
Adding RDMkit persistent identifiers

### PR DESCRIPTION
**Pull request by bedroesb**
This is temporarily using my fork of RDMkit, but this would be replaced with the real RDMkit domain if approved internally.

### CODE REVIEW

File: rdmkit/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+This namespace will redirect all urls from https://w3id.org/rdmkit/**ID** to https://bedroesb.github.io/rdmkit/**ID** 
+
+## Contact
+This space is administered by:  
+
+**Bert Droesbeke**  
+*Developer*  
+[ELIXIR Belgium](https://www.elixir-belgium.org/) 
+<bedro@psb.ugent.be>  
+GitHub: [bedroesb](https://github.com/bedroesb)
```
Comment (TallTed) [inline rdmkit/README.md:14]: preserve all intended line breaks with double-space at end of line
```suggestion
[ELIXIR Belgium](https://www.elixir-belgium.org/)  
<bedro@psb.ugent.be>  
GitHub: [bedroesb](https://github.com/bedroesb)  
```

**Comment by dgarijo** [comment]
@bedroesb can you pls add the suggestions by @TallTed ? They look good to me. Thanks!

**Comment by bedroesb** [comment]
yes! Sorry for my late response, after discussion we were thinking of doing the test setup under https://github.com/perma-id/w3id.org/tree/master/people and add the real domain later when we have everything sorted out on the RDMkit side (the real one)

File: people/bedroesb/rdmkit/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# /rdmkit/
+This [W3ID](https://w3id.org) provides a persistent URI namespace for RDMkit resources.
+
+## Uses
+This namespace will redirect all urls from https://w3id.org/rdmkit/**ID** to https://bedroesb.github.io/rdmkit/**ID**
```
Comment (dgarijo) [inline people/bedroesb/rdmkit/README.md:5]: sorry, but this is not accurate, is it? The w3id would be w3id.org/people/bedroesb/rdmkit/id

**Comment by dgarijo** [comment]
Sorry, see my comment in the PR. I am a little confused on the final w3id you are expecting here.

**Comment by bedroesb** [comment]
you are absolutely right! By moving to a test location I forgot to update the sentence in the README

**Comment by bedroesb** [comment]
thanks a lot everyone!

**Pull request merged**